### PR TITLE
Basic styling

### PR DESCRIPTION
**Pull request by codingoak**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,47 +23,51 @@ <h1>Daniel Eicher</h1>
   </header>
   <figure>
     <img src="https://source.unsplash.com/random/600x300" alt="">
-    <figcaption>This is an example image</figcaption>
+    <figcaption>Here will be be a picture of mysfelf</figcaption>
```
Comment (mwarnck) [inline index.html]: here is a little typo in the word "myself" ;)

File: index.html [hunk 2]
```diff
@@ -22,47 +23,51 @@ <h1>Daniel Eicher</h1>
...
   <section id="hobbies">
-    <h2>Hobbies:</h2>
+    <h2>Hobbies</h2>
     <ul>
       <li>Hiking</li>
       <li>Coding</li>
       <li>Reading books</li>
     </ul>
   </section>
   <section id="contact">
```
Comment (mwarnck) [inline index.html]: are interests and hobbies not the same? :) but code looks right in my opinion.

File: index.html [hunk 3]
```diff
@@ -22,47 +23,51 @@ <h1>Daniel Eicher</h1>
...
+            <a class="logo" href="https://www.facebook.com/daniel.eicher.355/"><img
+                src="https://upload.wikimedia.org/wikipedia/commons/5/51/Facebook_f_logo_%282019%29.svg" width="32px"
+                height="32px" alt="linkedin-logo"></a>
       </li>
     </ul>
   </section>
   <section id="biography">
-    <h2>Biography:</h2>
+    <h2>Biography</h2>
     <div>
```
Comment (mwarnck) [inline index.html:65]: Maybe I oversee this but I can not find the closing tag for this <div> element...

**Review by mwarnck — COMMENTED** [review]
overall looks good but have a look at my little comments ;)

File: index.html (hunk 1 above)
Comment (codingoak) [reply index.html]: thx

File: index.html (hunk 2 above)
Comment (codingoak) [reply index.html]: yes you're right :) i will change it soon.

**Pull request merged**